v3/src/components/graph/adornments/adornment-models.ts

```diff
@@ -4,7 +4,8 @@
 
 import {Instance, types} from "mobx-state-tree"
 import {typedId} from "../../../utilities/js-utils"
-import {Point, kMovableLineType} from "../graphing-types"
+import {Point, kMovableLineType, kMovablePointType} from "../graphing-types"
+import { kMovableValueType } from "./movable-value-types"
```

Comment:
Eventually, we'll want to remove these adornment-specific imports from this file by defining the union from the registration system lazily, but that can wait for a later PR.